packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -44,6 +44,8 @@ class ServiceConnectionManager {
     _serviceExtensionManager = ServiceExtensionManager(isolateManager);
   }
 
+  final ResolvedUriManager resolvedUriManager = ResolvedUriManager();
```

Comment:
nit: remove type on LHS and group below with other "managers" (`vmFlagManager`, `timelineStreamManager`, etc.)